### ISSUE TITLE
Sorting by impacts on selected items (need your opinion)

### ISSUE THREAD

**Issue by Danmer** [Danmer/vs-build-planner#57]
I added a new experimental feature: sorting weapons, passives and arcanas by impacts on selected items. So the most effective ones are going to be the first in the lists. There are some incorrect moments in counting impacts, especially arcana impacts on passives. It's quite complicated to handle this automatically. But even this state of sorting looks interesting in my opinion.

It's turned off by default and can be enabled in the settings.
Is it worth to be enabled by default or it's more confusing than helpful?

**Comment by wraith309** [comment]
I can certainly see the reasoning, to get a general sense of what items are most beneficial depending on what build you have so far, and I might turn it on when theorycrafting, but I still think it would be best to leave it off by default. it seems like it would be confusing and it would make finding specific items harder to do.

**Comment by wraith309** [comment]
no lie though. i turned it on when constructing a build i was already using, and it reminded me of other synergies i hadn't considered, so it's definitely a helpful feature.

**Comment by uqs** [comment]
Yeah, it's dope, but no comment on default on/off, it's fine either way.

**Comment by luismartinezs** [comment]
very nice feature, whether it's toggle on or off by default doesn't matter I think
